data/windows.yaml

```diff
@@ -2768,6 +2768,20 @@ sources:
 labels: [Users]
 supported_os: [Windows]
 ---
+name: WindowsUserJumpLists
```

Comment:
Please make this a group and split the different jumplist types into separate definitions 

This allows additions of other (lesser known) jump list formats, such as Explorer ProgramsCache Registry values. Also see: https://github.com/libyal/dtformats/blob/master/documentation/Jump%20lists%20format.asciidoc

Reply:
This should be fixed